src/probnum/filtsmooth/__init__.py

```diff
@@ -1,8 +1,19 @@
 """Bayesian Filtering and Smoothing."""
 
-from .bayesfiltsmooth import *
-from .filtsmoothposterior import *
-from .gaussfiltsmooth import *
+from .bayesfiltsmooth import BayesFiltSmooth
+from .filtsmoothposterior import FiltSmoothPosterior
+from .gaussfiltsmooth import (
```

Comment:
Actually, I think you forgot to replace the `*`-imports in `filtsmooth.gaussfiltsmooth.__init__.py`

Reply:
hmmmmm seems it. I thought I had... thanks for the pointer!